tests/containers/bci_test.pm

```diff
@@ -99,6 +99,9 @@ sub run {
     my $engine = $args->{runtime};
     my $bci_devel_repo = get_var('BCI_DEVEL_REPO');
     my $bci_tests_repo = get_required_var('BCI_TESTS_REPO');
+    if (my $bci_repo = get_var('REPO_BCI')) {
+        $bci_tests_repo = "http://openqa.suse.de/assets/repo/$bci_repo";
```

Comment:
Judging from the test module, this variable is actually never being used. I see the VRs are passing but are we sure, that they are using the desired BCI repo? :thinking:

Reply:
Right, good catch! Thanks

http://kepler.suse.cz/tests/21329#step/bci_test_podman/40